deployment/tfhub/object-detection-url.ipynb

```diff
@@ -0,0 +1,607 @@
+        "autoscaling = Autoscaling(min_replicas=1, max_replicas=20, min_scale=0.1, max_scale=10)\n",
+        "autoscaling.add_metric(CpuUtilizationTarget(0.6))\n",
+        "autoscaling.add_metric(MemoryUtilizationTarget(0.7))\n",
+        "autoscaling.add_metric(RequestsPerWorkerTarget(1))\n",
+        "resources = Resources(cpu=1., memory='4Gi')"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Y0gm6fOse6BP"
+      },
+      "source": [
+        "## 7. Model Endpoint"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "wZi8X-LoCORa"
+      },
+      "outputs": [],
+      "source": [
+        "endpoint = client.get_or_create_endpoint('object-detection-url')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "F4-9M2_xKF7R",
+        "outputId": "acf0965c-da8b-442b-f170-a9af8c2255bf"
+      },
+      "outputs": [],
+      "source": [
+        "%%time\n",
+        "endpoint.update(\n",
+        "    model, \n",
+        "    strategy = DirectUpdateStrategy(),\n",
+        "    autoscaling = autoscaling,\n",
+        "    resources = resources,\n",
+        "    wait = True\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "0Zgc18ocXbCE"
+      },
+      "outputs": [],
+      "source": [
+        "deployed_model = endpoint.get_deployed_model()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "NbxYiJJsX5oL"
+      },
+      "source": [
+        "## 8. Predictions"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "p7x7dfWoCNxK"
+      },
+      "outputs": [],
+      "source": [
+        "url = 'http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/'"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "4nS6rzzyK58c"
+      },
+      "outputs": [],
+      "source": [
+        "req = requests.get(url)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "gH_5iPC84jmu"
+      },
+      "outputs": [],
+      "source": [
+        "soup = BeautifulSoup(req.text, 'lxml')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Hl_MzSFXX2oo"
+      },
+      "outputs": [],
+      "source": [
+        "urls = []\n",
+        "\n",
+        "for link in soup.find_all('a'):\n",
+        "    href = link.get('href')\n",
+        "  \n",
+        "    if href.endswith('.jpg'):\n",
+        "        url = f\"http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/{href}\"\n",
```

Comment:
Since I'm requesting changes anyway: we should try reusing the

```
url = 'http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/'
```

from the cell above? Because we are basically reusing the `http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/` part here.

And we should do this in the other notebook, also.

Reply:
Also we should try to avoid setting the `url` variable for two different values like this.

Reply:
Done.